### DISCUSSION TITLE
[question] how do I make eyes brighter?

### DISCUSSION THREAD

**Discussion by throwaway242685** [tannerhelland/PhotoDemon#535] [Q&A]
hi, how can I make eyes brighter on a picture with PhotoDemon?

**Comment by hi5** [comment]
That is too broad a question. There are various filters you could try to enhance the entire image, including the eyes. The "Adjustment, Auto correct or Auto Enhance" might be all that you need. 

You could try to select the eyes using the ellipse/freehand tool and then apply the filter(s) to selected area. Brightness & Contrast is another one to try.

This is "select eyes and Adjustment, Auto correct (2x)" as an example (before/after):
![eyes](https://github.com/tannerhelland/PhotoDemon/assets/1973899/c35e2016-ccde-4eef-a853-f9aae0d7f4dc)
